custom_components/adaptive_lighting/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "domain": "adaptive_lighting",
   "name": "Adaptive Lighting",
-  "documentation": "https://github.com/basnijholt/adaptive-lighting#readme",
-  "issue_tracker": "https://github.com/basnijholt/adaptive-lighting/issues",
+  "codeowners": ["@basnijholt", "@RubenKelevra", "@th3w1zard1"],
   "config_flow": true,
   "dependencies": [],
-  "codeowners": ["@basnijholt", "@RubenKelevra"],
-  "version": "1.4.1",
+  "documentation": "https://github.com/basnijholt/adaptive-lighting#readme",
+  "iot_class": "calculated",
+  "issue_tracker": "https://github.com/basnijholt/adaptive-lighting/issues",
```

Comment:
The error seems to indicate that the problem is on character three of this line. However, I do not understand what the issue is?

Reply:
@basnijholt see https://github.com/basnijholt/adaptive-lighting/pull/467/commits/845c15015927fdeef4464eb1dac773ad42e6aa00 when I messed up the comma, it broke hass. when I fixed the comma, it broke HACS.
Something ain't updating somewhere, maybe there's a cache and it'll figure itself out over time.

Reply:
For reference, when I broke _**HASS validation**_ it was due to
```
{
  "domain": "adaptive_lighting",
  "name": "Adaptive Lighting",
  "codeowners": ["@basnijholt", "@RubenKelevra"],
  "config_flow": true,
  "dependencies": [],
  "documentation": "https://github.com/basnijholt/adaptive-lighting#readme",
  "iot_class": "calculated"
  "issue_tracker": "https://github.com/basnijholt/adaptive-lighting/issues",
  "requirements": [],
  "version": "1.4.1",
}
```
which outputted `line 9 column 3 (char 268)`

Fixing the comma in https://github.com/basnijholt/adaptive-lighting/commit/845c15015927fdeef4464eb1dac773ad42e6aa00 outputs the same error.

Reply:
I'm going to try opening up another pull request with fresh commits to see if anything magical happens.

Reply:
I’ve just merged it because it already fixes a real problem. Then in the next PR hopefully there is no caching issue. If that was the problem even 😜